### ISSUE TITLE
increased/decreased circulating glucose AUC

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3581]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

A. increased circulating glucose AUC
B. decreased circulating glucose AUC

>2. Synonyms (e.g., absent spleen)

A. increased circulating glucose area under the curve
B. decreased circulating glucose area under the curve

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

A. greater than the normal concentration in the blood of this major monosaccharide of the body over a specified time of measurement
B. less than the normal concentration in the blood of this major monosaccharide of the body over a specified time of measurement

PMID:10323602

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

A. increased circulating glucose level (MP:0005559)
B. decreased circulating glucose level (MP:0005560)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5256-8683

**Comment by sbello** [comment]
I think we don't want to add this to the MP. While we have some measurement specific terms these are generally where the measurement is the phenotype.
